src/screens/Login/LoginScreen.tsx

```diff
@@ -61,21 +61,36 @@ const LoginScreen: React.FC<LoginScreenProps> = ({ setCurrentUser }) => {
   useEffect(() => {
     if (!loading && !error && loginData) {
       const { login } = loginData;
-      (async () =>
+      async () => {
```

Comment:
This doesn't get executed. It's just declaring an async function. It needs to be an immediately invoked function. It needs the wrapper brackets